tests/core/document/test_documents_common.py

```diff
@@ -39,4 +43,63 @@ def test_translate_vespa_document_response_logging(self):
         with patch("marqo.core.document.document.logger.error") as mock_log_error:
             status = 400
             self.document.translate_vespa_document_response(status)
-        mock_log_error.assert_called_once()
\ No newline at end of file
+        mock_log_error.assert_called_once()
+
+    def test_marqoAddDocumentsResponseDictFormat(self):
+        item = MarqoAddDocumentsItem(status=200, id="id", message="message", error="error", code="code")
+        res = MarqoAddDocumentsResponse(errors=False, processingTimeMs=0.0, index_name="index_name", items=[item])
+
+        returned_dict = res.dict(exclude_none=True, by_alias=True)
+        expected_dict = {
+            "errors": False,
+            "processingTimeMs": 0.0,
+            "index_name": "index_name",
+            "items": [
+                {
+                    "status": 200,
+                    "_id": "id",
+                    "message": "message",
+                    "error": "error",
+                    "code": "code"
+                }
+            ]
+        }
+        self.assertEqual(expected_dict, returned_dict)
+
+    def test_marqoUpdateDocumentsResponseDictFormat(self):
+        item = MarqoUpdateDocumentsItem(id="id", status=200, message="message", error="error")
+        res = MarqoUpdateDocumentsResponse(errors=False, index_name="index_name", items=[item], processingTimeMs=0.0)
+
+        returned_dict = res.dict(exclude_none=True, by_alias=True)
+        expected_dict = {
+            "errors": False,
+            "index_name": "index_name",
+            "items": [
+                {
+                    "_id": "id",
+                    "status": 200,
+                    "message": "message",
+                    "error": "error"
+                }
+            ],
+            "processingTimeMs": 0.0
+        }
+        self.assertEqual(expected_dict, returned_dict)
+
+    def test_marqoGetDocumentsByIdsResponseDictFormat(self):
```

Comment:
Hard to understand what we're testing. A short description would help for each test

Reply:
Done